config.tfvars

```diff
@@ -4,23 +4,37 @@
 
 # Please define the values to configure the infrastructure before install
 
+################################################################################
+# Mandatory
+################################################################################
+
 # 'environment_name' provides your environment a unique name within a single cloud provider account.
 # This value can not be altered after the configuration has been applied.
 environment_name = "<ENVIRONMENT>"
 
 # Cloud provider region that this configuration will deploy to.
 region = "<REGION>"
 
-# Custom tags for all resources to be created. Please add all tags you need to propagate among the resources.
-resource_tags = {
-  Terraform = "true"
-}
+# Bamboo license
+# To avoid storing license in a plain text file, we recommend storing it in an environment variable prefixed with `TF_VAR_` (i.e. `TF_VAR_bamboo_license`) and keep the below line commented out
+# If storing license as plain-text is not a concern for this environment, feel free to uncomment the following line and supply the license here
+#
+#bamboo_license = "<license key>"
 
-# Instance types that is preferred for node group.
-instance_types = ["m5.4xlarge"]
+# Bamboo system admin credentials
+# WARNING: In case you are restoring an existing dataset (see the `dataset_url` property below), you will need to use
+# credentials existing in the dataset. In this case, the `bamboo_admin_*` properties below are ignored.
```

Comment:
We had some change here as part of TFKUBE-234 which is not here. Would you please pull the latest changes in main?